src/Type.cpp

```diff
@@ -71,7 +72,20 @@ Type *Type::void_type = NULL;
 static vector<Type *> AllTypes;
 static vector<Type *> derived_types;
 
+AttributeGenerator type_attr_generator;
+
 //////////////////////////////////////////////////////////////////////
+
+void
+TypeAttributes()
```

Comment:
Name it please somehow `InitializeTypeAttributes` or similar.

Reply:
Sure.